go/vt/sqlparser/parse_test.go

```diff
@@ -2561,6 +2561,18 @@ var (
 		}, {
 			input:  "CREATE PROCEDURE proc (IN p_store_id INT) SELECT * FROM inventory WHERE store_id = p_store_id INTO DUMPFILE 'dumpfile.txt'",
 			output: "create procedure proc (in p_store_id INT) select * from inventory where store_id = p_store_id into dumpfile 'dumpfile.txt'",
+		}, {
+			input:  "CREATE TABLE t (id INT PRIMARY KEY, col1 GEOMETRY SRID 0)",
+			output: "create table t (\n\tid INT primary key,\n\tcol1 GEOMETRY srid 0\n)",
+		}, {
+			input:  "CREATE TABLE t (id INT PRIMARY KEY, col1 GEOMETRY NOT NULL SRID 0)",
+			output: "create table t (\n\tid INT primary key,\n\tcol1 GEOMETRY not null srid 0\n)",
+		}, {
+			input:  "ALTER TABLE t ADD COLUMN col1 POINT NOT NULL DEFAULT (POINT(1, 2)) SRID 0",
+			output: "alter table t add column (\n\tcol1 POINT not null default (POINT(1, 2)) srid 0\n)",
+		}, {
+			input:  "ALTER TABLE t MODIFY COLUMN col1 POINT NOT NULL DEFAULT (POINT(1, 2)) SRID 1234",
```

Comment:
(minor) might be nice to use another geo type, like `POLYGON` in one of these tests, just to add some more variety beyond point and geometry.